### ISSUE TITLE
Implement Batch normalisation to prevent Bias shift

### ISSUE THREAD

**Issue by Jonathan-Rasch** [JonathanLR/tinyTensor#2]
[Bias shift](https://www.quora.com/Why-do-non-zero-mean-activations-induce-a-bias-shift-for-units-in-the-next-layer-and-why-is-that-bad)

[batch normalisation](https://medium.com/deeper-learning/glossary-of-deep-learning-batch-normalisation-8266dcd2fa82)
